src/autoscaler-common/types.js

```diff
@@ -25,6 +25,7 @@ const AutoscalerUnits = {
  *    currentSize: number,
  *    regional: boolean,
  *    currentNodes: number,
+ *    numDatabases: number,
```

Comment:
Worth considering `currentDatabases`, `currentNumDatabases`, or similar, to be consistent with the other metadata fields that may change over time?

Reply:
Done